<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>coda_stampa_window</class>
 <widget class="QMainWindow" name="coda_stampa_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coda di stampa</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>781</width>
     <height>576</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="lbl_title">
      <property name="font">
       <font>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Coda di stampa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="table_coda_stampa">
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Username</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome file</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Data e ora</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tipo di carta</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Tipo di rilegatura</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lbl_num_docs">
        <property name="text">
         <string>Ci sono &lt;num_docs&gt; documenti nella coda di stampa.</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_close">
        <property name="text">
         <string>Chiudi</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
